decidim-proposals/app/controllers/concerns/decidim/proposals/orderable.rb

```diff
@@ -56,6 +56,8 @@ def reorder(proposals)
             proposals.order(proposal_votes_count: :desc)
           when "recent"
             proposals.order(created_at: :desc)
+          else
+            proposals
```

Comment:
My idea was that `default_order` never returns nil, and it's use as a fallback for the `detect_order` method. That's why I set the order like this:

```ruby
def order
  @order ||= detect_order(params[:order]) || default_order
end
```

I think it's enough to fix the bug I introduced by ensuring `default_order` never returns `nil`.